src/views/templates/list/components/VirtualMachineTemplatesColumns.tsx

```diff
@@ -0,0 +1,34 @@
+import { TFunction } from 'i18next';
+
+import { K8sResourceCommon, TableColumn } from '@openshift-console/dynamic-plugin-sdk';
+import { sortable } from '@patternfly/react-table';
+
+const VirtualMachineTemplatesColumns: (t: TFunction) => TableColumn<K8sResourceCommon>[] = (t) => [
```

Comment:
it can be exported as custom hook so you don't need to pass t into it and just consume t :)